src/databases/models/products.ts

```diff
@@ -17,7 +17,7 @@ class Products extends Model<IProduct> {
     declare bonus?: string;
     declare images: string[];
     declare quantity: number;
-    declare isAvailable: string;
+    declare status: string;
     declare createdAt: Date;
     declare updatedAt: Date;
 

```

Comment:
Why do we have this relation (  **`Products.belongsTo(models.Users, { foreignKey: "userId", as: "user" });`** ) while we are not using **`userId`** in our product table ?